### DISCUSSION TITLE
Are you open to design contributions?

### DISCUSSION THREAD

**Discussion by fivenp** [graphql/graphiql#2946] [Ideas]
Hey there!

I'm a long time GraphiQL user myself and always thought about contribution some ux and design expertise to it – but couldn't find the right way to do so (yet). (Hard to understand where to start and find documentation, resources, etc on UX topics)

Talking to a lot of other designers they seem to face the same issue very often. (also on other OSS projects)

Therefor I've built https://contribute.design/ which helps designers and open source software to understand each other better. Basically I'm trying to help projects get the guidelines for designers set up and create some visibility across designers for the project.

My question now is – would you be open to participate in this initiative and can I potentially support you in any way here? I'd really love to enable other designers to participate in amazing tools like GraphiQL :)

**Comment by jonathanawesome** [comment]
Hi 👋! There are a couple of recent and ongoing design-related discussions that you could check out (#2216 , #2788).

**Reply by fivenp** [reply]
Yeah – I saw them – and those are amazing discussions to be honest. I was curious if it would make sense to bundle some of these resources and knowledge into some simple contribution guidelines for other designers to help them be more efficient at jumping in and potentially participate....

Maybe I'll prep a PR with what I mean and you folks can jump in? :)
